Modules/WorldBuffModule.cs

```diff
@@ -109,7 +109,7 @@ public class WorldBuffModule : ModuleBase<SocketCommandContext>
                 // Saving the buff instance
                 await context.AddAsync(buff);
                 await context.SaveChangesAsync();
-                await Context.Message.AddReactionAsync(Emote.Parse(_icons[buff.Type]));
+                // await Context.Message.AddReactionAsync(Emote.Parse(_icons[buff.Type]));
```

Comment:
This can be uncommented, commited by accident and commented it out since it doesn't work on my local.